lua/fzf-lua-frecency/fn_transform.lua

```diff
@@ -37,12 +37,9 @@ M.get_fn_transform = function(rpc_opts)
 
     -- only call fs_stat on files from the db, fd/rg files are guaranteed to exist
     if rpc_opts.stat_file and date_at_score_one then
-      if not vim.uv.fs_stat(abs_file) then
```

Comment:
Removing the entry here doesn't hurt, but I think I prefer to do all the filtering in `algo`

Reply:
Any score update with `update_file_score` will iterate over all the frecency files in order to re-sort them with the latest scores. `fs_stat` filtering also happens during this step, so every `update_file_score` effectively forces a big refilter

Reply:
You're right though, the entry will stay in the db for the rest of the `frecency` function call, but it'll get removed as soon as a file is selected / any other `BufWinEnter`